USER_GUIDE.md

```diff
@@ -228,6 +230,33 @@ print('\n The deleted point in time:')
 print(response)
 ```
 
+## Using High-level Python client
```

Comment:
Is this the only piece that is being carried forward from the high level client?

Reply:
All other API's added have similar implementation. So, I added only Search API example.